pages/translations/ar/LC_MESSAGES/messages.po

```diff
@@ -3543,58 +3110,58 @@ msgid "AMP Conf 2020: Save the date!"
 msgstr "مؤتمر AMP لعام 2020: احفظ التاريخ!"
 
 msgid "AMP is a web component framework to easily create user-first experiences for the web."
-msgstr ""
+msgstr "AMP هو إطار عمل لمكونات الويب يسهّل إنشاء تجارب للويب مع وضع المستخدم على رأس الأولويات."
 
 msgid "Easily create websites with a great page experience out-of-the-box using AMP’s ready-to-go web components."
-msgstr ""
+msgstr "يمكنك إنشاء مواقع الويب بسهولة بتجربة صفحة رائعة وحديثة باستخدام مكونات الويب الجاهزة للاستخدام من AMP."
 
 msgid "All about AMP Websites"
-msgstr ""
+msgstr "كل ما يتعلق بمواقع AMP على الويب"
 
 msgid "Get started"
-msgstr ""
+msgstr "بدء الاستخدام"
 
 msgid "Immerse readers in visual and tappable stories they can share anywhere on the open web."
-msgstr ""
+msgstr "اغمر القراء بالقصص المرئية والقابلة للنقر التي يمكنهم مشاركتها في أي مكان على الويب المفتوح."
 
 msgid "All about Web Stories"
-msgstr ""
+msgstr "كل ما يتعلق بقصص الويب"
 
 msgid "Serve lightning fast and secure eye-catching creatives."
-msgstr ""
+msgstr "تقديم تصاميم إبداعية بسرعة البرق وآمنة وملفتة للنظر."
 
 msgid "Send emails of the future with personalized dynamic content and interactive elements that engage readers."
-msgstr ""
+msgstr "إرسال رسائل بريد إلكتروني مستقبلية تحتوي على محتوى ديناميكي مخصص وعناصر تفاعلية تجذب القراء."
 
 msgid "The latest news"
-msgstr ""
+msgstr "آخر الأخبار"
 
 msgid "Don't want to miss any of the great stuff that happens around AMP? Then always be first in line and subscribe to our newsletter!"
-msgstr ""
+msgstr "ألا ترغب في عدم تفويت أي من الأمور الرائعة التي تحدث في إطار استخدام AMP؟ فكن الأول دائمًا واشترك في نشرتنا الإخبارية!"
 
 msgid "Don’t take our word for it - read case studies from industry success stories and see how AMP has produced positive results."
-msgstr ""
+msgstr "لا تأخذ كلامنا على محمل الجد - اقرأ دراسات الحالة من قصص النجاح في المجال وشاهد كيف حقق AMP نتائج إيجابية."
 
 msgid "A framework that amplifies page experience."
-msgstr ""
+msgstr "إطار عمل يعزز تجربة الصفحة."
 
 msgid "AMP is a simple and robust web components framework that makes it easy to build fast websites that are user-first and makes money. AMP provides long-term success for your web strategy with distribution across popular platforms and reduced operating costs."
-msgstr ""
+msgstr "يعدّ AMP إطار عمل بسيطًا وقويًا لمكونات الويب يسهل بناء مواقع الويب الخاصة بك التي تضع المستخدم على رأس الأولويات وتحقق الأرباح. يوفر AMP نجاحًا طويل الأمد لاستراتيجية الويب التي تستخدمها بفضل التوزيع على مستوى المنصات الرائجة وبتكاليف تشغيل وتطوير منخفضة."
 
 msgid "Guaranteed Page Experience"
```

Comment:
@nainar @sebastianbenz should "Page Experience" be localized?

Reply:
Yes, it's also translated in the official guidance, e.g. here: https://developers.google.com/search/docs/guides/page-experience. Best approach would be to look at the translation provided here for the "official" translation.